hepcrawl/spiders/desy_spider.py

```diff
@@ -297,22 +258,24 @@ def _parsed_items_from_marcxml(
                     record = marcxml2record(xml_record)
                     parsed_item = ParsedItem(record=record, record_format='hep')
                     parsed_item.file_name = file_name
-                    if not self.s3_enabled:
-                        files_to_download = [
-                            self._get_full_uri(
-                                current_url=document['url'],
-                                base_url=base_url,
-                            )
-                            for document in parsed_item.record.get('documents', [])
-                            if self._has_to_be_downloaded(document['url'])
-                        ]
-                    else:
-                        files_to_download = []
+                    new_documents = []
+                    files_to_download = []
+                    self.logger.info("Parsed document: %s", parsed_item.record)
+                    self.logger.info("Record have documents: %s", "documents" in parsed_item.record)
+                    for document in parsed_item.record.get('documents'):
+                        if self._is_local_path(document['url']):
+                            document['url'] = self._get_full_uri(document['url'])
+                            new_documents.append(document)
+                            self.logger.info("Updating document %s", document)
+                        else:
+                            files_to_download.append(document['url'])
+
+                    if new_documents:
+                        parsed_item.record['documents'] = new_documents
 
                     parsed_item.file_urls = files_to_download
-
                     self.logger.info('Got the following attached documents to download: %s' % files_to_download)
-                    self.logger.info('Got item: %s' % parsed_item)
+                    self.logger.info('Got item: %s', parsed_item)
```

Comment:
why only here :)

Reply:
I noticed it only here. :P I will look for more.

Reply:
Does it matter :)? I mean just leave it as is, anyway we have it everywhere.

Reply:
In logger module it's recommended to use it's own formatter.